<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>211</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_WCZYTAJ">
       <property name="text">
        <string>WCZYTAJ</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_WCZYTAJ"/>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ZAPISZ">
       <property name="text">
        <string>ZAPISZ</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_ZAPISZ"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_STATUS">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>WCZYTAJ PLIK</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_SCIEZKA">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>470</y>
      <width>731</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>SCIEZKA</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_AMPLITUDA">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>102</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AMPLITUDA">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ZMIANA AMPLITUDY</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_AMPLITUDA">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>100</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>MODYFIKATOR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_INWERSJA">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>150</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>INWERSJA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_WYCISZ">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>WYCISZ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_WYCISZ_OD">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>201</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>OD</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_WYCISZ_OD">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>205</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_WYCISZ_DO">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>201</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>DO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_WYCISZ_DO">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>209</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ZWIEKSZ_GLOSNOSC">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>250</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ZWIĘKSZ GŁOSNOSC</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ZWIEKSZ_GLOSNOSC">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>250</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>MODYFIKATOR</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ZWIEKSZ_GLOSNOSC">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>259</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ZMNIEJSZ_GLOSNOSC">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>300</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ZMNIEJSZ GŁOSNOSC</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ZMNIEJSZ_GLOSNOSC">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>300</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>MODYFIKATOR</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ZMNIEJSZ_GLOSNOSC">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>310</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Export_Do_Txt">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>350</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>EXPORT DO TXT</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Eksport_Do_Txt">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>360</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Eksport_Do_Txt">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>340</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>SCIEŻKA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ECHO">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>400</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ILOSC PROBEK</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProcesorySygnalowe">
    <property name="title">
     <string>ProcesorySygnalowe</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuRemigiuszWojewodzki">
    <property name="title">
     <string>RemigiuszWojewodzki</string>
    </property>
    <addaction name="action241516"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuProcesorySygnalowe"/>
   <addaction name="menuRemigiuszWojewodzki"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action241516">
   <property name="text">
    <string>241516</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
